.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 install:
 - npm install
 - pip install --user -r requirements.txt
-- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="gcc-4.9"; fi
+- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="g++-4.9 -std=c++11"; fi
```

Comment:
I don't know about these env with respect to emscripten. 

Are these used by emscripten or only native compilers? Regardless, we should keep these identical in our local machines (same OS) and travis. If so, should these changes go into make files also?

Also, if these are used by emscripten, does this increase size of the runtime with C++11?

Besides should we refrain from switching to "C++11" for portability (of Vireo) reasons?